### ISSUE TITLE
running SEDT on custom data

### ISSUE THREAD

**Issue by Lou1sM** [Anaesthesiaye/sound_event_detection_transformer#5]
Hi,

I'm trying to run SEDT on my own data, but during fine-tuning the loss just bounces around randomly, and during inference it predicts no events at all. I'm using the checkpoints you provide and have tried lots of tweaking of training parameters but it never learns anything. Maybe my data (Raven selection tables) is formatted differently to the data your model expects, or something is going wrong during the various steps of assembling the dataset, but I can't figure out what. 

Would you be able to provide a short example of how to run SEDT on custom data? Even just how to execute a single forward pass at inference to generate predictions. Thanks in advance.
